org/custom_model_field.py

```diff
@@ -47,18 +47,18 @@ def permissions_to_integer(self):
         return permission_integer
 
 
-    def get_permission_list(self, permission_int):
-        """Returns a list of permissions available"""
-
-        permissions_list = [
-            self.IS_ADMIN,
-            self.IS_STAFF,
-            self.CAN_CREATE_TASKS,
-            self.CAN_REPLY_TO_QUERIES,
-            self.CAN_REVIEW_PROOFS
-        ]
-
-        return permissions_list
+    def get_permission_dict(self):
```

Comment:
```suggestion
    def get_permission_list(self):
```
Also instead of returning `Permission for task creation`, `Admin Permission` etc., Please remove the underscores and titlecase the permission names :innocent:

Reply:
I did not understand 😅 
what do I do instead of returning `Permission for task creation`  et cetera...? 

I remember that I was supposed to return a dictionary instead of a list in `org.custom_model_field.PermissionSet.get_permission_list` with keys as user friendly names, that was mentioned in the issue, shall I change it to a list..? Or am I misinterpreting this too..? 😅

Reply:
1. return this in the `get_permission_list()` 
1. and with help of `get_permission_list()` return the same in the API.
```
{
    "Is admin" : self.IS_ADMIN,
    "Can create tasks": self.CAN_CREATE_TASKS,
    # et cetera...
}
```

Reply:
Ahh got it 👍👍